edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/appsettings_hub.json

```diff
@@ -60,16 +60,19 @@
   "storeAndForward": {
     "timeToLiveSecs": "-1"
   },
+  "metrics": {
+    "enabled": true,
+    "listener": {
+      "port": 80,
```

Comment:
We should probably try to allocate a port here: https://github.com/prometheus/prometheus/wiki/Default-port-allocations and then default to it. For now, we can start with something around `9600`

Reply:
cool, sounds good